addons/point_of_sale/upgrades/1.0.2/post-deduplicate-uuids.py

```diff
@@ -28,10 +30,22 @@ def deduplicate_uuids(table):
             cr.execute(query)
             if not cr.rowcount:
                 break
-            ids = [r[0] for r in cr.fetchmany(100000)]
+            ids_to_uuids_as_str = ', '.join([
+                    f"({r[0]}, '{uuid.uuid4()!s}')"
+                    for r in cr.fetchmany(BULK_SIZE)
+            ])
             cr.execute(
-                f"UPDATE {table} SET uuid = (%s::json)->>(id::text) WHERE id IN %s",
-                [Json({id_: str(uuid.uuid4()) for id_ in ids}), tuple(ids)]
+                """
+                WITH values AS (
```

Comment:
`values` is a reserved keyword in Postgres, better use something else

```suggestion
                WITH vals AS (
```